lib/cvsdb.py

```diff
@@ -393,7 +402,7 @@ def SQLQueryListString(self, field, query_entry_list):
             elif query_entry.match == "notregex":
                 match = " NOT REGEXP "
 
-            sqlList.append("%s%s%s" % (field, match, self.db.literal(data).decode("utf-8")))
+            sqlList.append("%s%s%s" % (field, match, self.db.literal(data)))
```

Comment:
The doc string of [pymysql.connections.connection.literal()](https://github.com/PyMySQL/PyMySQL/blob/main/pymysql/connections.py#L511) says that it is not used in normal application, and it seems that its behaviour is different between in MySQLdb and in pymysql. So I think we should rewrite it without literal().